ui/app/templates/components/clients/config.hbs

```diff
@@ -1,80 +1,38 @@
 {{#if (eq @mode "edit")}}
-  <form onsubmit={{action "onSaveChanges"}} data-test-pricing-metrics-config-form>
+  <form onsubmit={{action "onSaveChanges"}} data-test-clients-config-form>
     <div class="box is-sideless is-fullwidth is-marginless">
       <MessageError @model={{@model}} @errorMessage={{this.error}} />
-      {{#each @model.configAttrs as |attr|}}
-        {{#if (and (eq attr.type "string") (eq attr.options.editType "boolean"))}}
+      {{#each @model.formFields as |attr|}}
+        {{#if (eq attr.name "enabled")}}
           <label class="is-label">Usage data collection</label>
-          {{#if attr.options.helpText}}
-            <p class="sub-text">
-              {{attr.options.helpText}}
-              {{#if attr.options.docLink}}
-                <DocLink @path={{attr.options.docLink}}>
-                  See our documentation
-                </DocLink>
-                for help.
-              {{/if}}
-            </p>
-          {{/if}}
+          <p class="sub-text">
+            Enable or disable client tracking. Keep in mind that disabling tracking will delete the data for the current
+            month.
+          </p>
           <div class="control is-flex has-bottom-margin-l">
             <input
-              data-test-field
+              data-test-input="enabled"
               type="checkbox"
-              id={{attr.name}}
-              name={{attr.name}}
+              id="enabled"
+              name="enabled"
               class="switch is-rounded is-success is-small"
-              checked={{eq (get @model attr.name) attr.options.trueValue}}
-              onchange={{action (action "updateBooleanValue" attr) value="target.checked"}}
+              disabled={{@model.reportingEnabled}}
+              checked={{eq @model.enabled "On"}}
+              onchange={{this.toggleEnabled}}
```

Comment:
should this be `{{on "change" this.toggleEnabled}}` or does it not really matter?

Reply:
Yes good call. I was updating some older patterns in this component and only partially updated this one.

> or does it not really matter?

That's a really good question. It works either way but I think it's best to follow the _Ember way_ and use the `on` modifier.